action.yml

```diff
@@ -3,16 +3,18 @@ description: 'GHA to build and publish a 🛢️ container'
 
 inputs:
   build_arch:
-    description: What arch to build for. Comma seperated list
+    description: What arch to build for. Comma seperated list.
     default: linux/amd64
+  build_args:
+    description: Build args to pass to the build. This is a string of key=value pairs seperated by a space.
```

Comment:
doesn't it need a default value?

Reply:
depends in the end on the dockerfile. if a ARG is empty, how it is handled, but for the action it is okay if its empty. and what default for a arg do you wanna set?

Reply:
I wasn't sure if users have to provide a value when there's no default. Then this would be a breaking change.

Reply:
no i tested this, it's okay to have this empty, for the action, what the Dockerfile/Containerfile does with this is another thing but out of our scope